plugin/validator_test.go

```diff
@@ -2378,3 +2378,63 @@ func getErrForField(e error, path string) string {
 	}
 	return ""
 }
+
+func TestValidator_Transformations(t *testing.T) {
```

Comment:
Can we assert that `v.Transform(plugin)` no-ops when the schema doesn't have any transformations?